Task 2/Word Count.js

```diff
@@ -0,0 +1,12 @@
+/*
+Question Link
+http://www.codewars.com/kata/word-count
+*/
+
+function countWords(myString) {
```

Comment:
The parameter `myString` can be renamed to a more meaningful name. Something like `inputString`.